superset/databases/dao.py

```diff
@@ -124,3 +125,13 @@ def get_related_objects(cls, database_id: int) -> Dict[str, Any]:
         return dict(
             charts=charts, dashboards=dashboards, sqllab_tab_states=sqllab_tab_states
         )
+
+    @classmethod
+    def get_ssh_tunnel(cls, database_id: int) -> Dict[str, Any]:
+        ssh_tunnel = (
+            db.session.query(SSHTunnel)
+            .filter(SSHTunnel.database_id == database_id)
+            .one_or_none()
+        )
+
+        return dict(ssh_tunnel=ssh_tunnel)
```

Comment:
why return a dictionary instead of the original model instance?

Reply:
Just wanted to wrap it in a dict since it's a property of the DB and not the DAO entity itself. But yeah, could just send the instance.